File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/Regelendring.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.kode.RegelendringTypeCti;

import java.io.Serializable;
import java.util.Date;

public class Regelendring implements Serializable {



    /**
     * Datoen en regel- eller satsendring har virkningsdato.
     */
    private Date endringsdato;

    /**
     * Tekst som beskriver typen endring, ref. kodeverk.
     */
    private RegelendringTypeCti endringstype;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Regelendring.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.enum.RegelendringTypeEnum
import java.io.Serializable
import java.util.*
import java.time.LocalDate

class Regelendring : Serializable {
    /**
     * Datoen en regel- eller satsendring har virkningsdato.
     */
    var endringsdatoLd: LocalDate? = null

    /**
     * Tekst som beskriver typen endring, ref. kodeverk.
     */
    var endringstypeEnum: RegelendringTypeEnum? = null

}